src/decay.c

```diff
@@ -3,6 +3,13 @@
 
 #include "jemalloc/internal/decay.h"
 
+const uint64_t h_steps[SMOOTHSTEP_NSTEPS] = {
```

Comment:
Love it. I think this can be static now, right?

Reply:
Correct!